sistema-companhia-aerea/src/main/java/br/edu/ifto/sistemacompanhiaaerea/repository/VooRepository.java

```diff
@@ -0,0 +1,30 @@
+package br.edu.ifto.sistemacompanhiaaerea.repository;
+
+import br.edu.ifto.sistemacompanhiaaerea.model.Voo;
+import br.edu.ifto.sistemacompanhiaaerea.model.CompanhiaAerea;
+import br.edu.ifto.sistemacompanhiaaerea.model.Aeroporto;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+import java.util.List;
+import java.util.Optional;
+
+
+@RepositoryRestResource (collectionResourceRel ="Voo",path = "Voo")
+public interface VooRepository extends PagingAndSortingRepository <Voo , Long> {
+//    @Override
+//  Optional<Voo> findById(@Param("Id")Long id);
```

Comment:
O findById é padrão na interface que vc estende. Deve remover.
Não deve-se deixar código comentado (exceto se ele for pra ser descomentado em algum momento, que não é o caso). Isso só polui o código.